static/api/extensions/dhenry/extension-stats/1.0.1/README.md

```
@@ -0,0 +1,174 @@
+   - The extension should load immediately
+
+See [INSTALL-SIMPLE.md](INSTALL-SIMPLE.md) for detailed instructions.
+
+### Method 2: OCI Image (Recommended for Users)
+
+Install directly from the published OCI image:
+
+1. Open Podman Desktop
+2. Go to **Extensions** → **Install custom...**
+3. Enter:
+   ```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)